concrete_datastore/concrete/constants.py

```diff
@@ -63,3 +64,14 @@
     'DateField': 'date',
     'BigIntegerField': 'int',
 }
+
+EMPTY_VALUES_MAP = {
+    'TextField': {'empty_value': '', 'field_type': str},
+    'CharField': {'empty_value': '', 'field_type': str},
+    'IntegerField': {'empty_value': 0, 'field_type': int},
+    'FloatField': {'empty_value': 0.0, 'field_type': float},
+    'DecimalField': {'empty_value': Decimal(0), 'field_type': Decimal},
+    'JSONField': {'empty_value': {}, 'field_type': dict},
+    'BigIntegerField': {'empty_value': 0, 'field_type': int},
+    'BooleanField': {'empty_value': None, 'field_type': bool},
+}
```

Comment:
Remove all and keep only TextField and Charfield. cf validators.py

Reply:
removed. The only keys kept are `CharField` `TextField`.